subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/ReportImplementation.kt

```diff
@@ -298,6 +304,10 @@ class ReportImplementation(
         addEntry(Entry.Check(assertionMessage, timeProvider.nowInSeconds()))
     }
 
+    override fun addImageSynchronously(file: File): String {
+        return screenshotTestImageUploader.uploadScreenshots(file, null)
```

Comment:
Do you have a use case when you give the second parameter not-null?

Reply:
second parameter is not required